packages/datadog-plugin-grpc/test/integration-test/server.mjs

```diff
@@ -0,0 +1,61 @@
+import 'dd-trace/init.js'
+import * as grpc from '@grpc/grpc-js'
+import * as protoLoader from '@grpc/proto-loader'
+import path from 'path'
+import getPort from 'get-port'
+
+const currentDirectoryPath = path.dirname(new URL(import.meta.url).pathname)
+const parentDirectoryPath = path.resolve(currentDirectoryPath, '..')
+
+let server
+const port = await getPort()
+
+function buildClient (service, callback) {
+  service = Object.assign(
+    {
+      getBidi: () => {},
+      getServerStream: () => {},
+      getClientStream: () => {},
+      getUnary: () => {}
+    },
+    service
+  )
+
+  const definition = protoLoader.loadSync(`${parentDirectoryPath}/test.proto`)
+  const TestService = grpc.loadPackageDefinition(definition).test.TestService
+
+  server = new grpc.Server()
+
+  return new Promise((resolve, reject) => {
+    if (server.bindAsync) {
+      server.bindAsync(`0.0.0.0:${port}`, grpc.ServerCredentials.createInsecure(), (err) => {
+        if (err) return reject(err)
+
+        server.addService(TestService.service, service)
+        server.start()
+
+        resolve(new TestService(`localhost:${port}`, grpc.credentials.createInsecure()))
+      })
+    } else {
+      server.bind(`0.0.0.0:${port}`, grpc.ServerCredentials.createInsecure())
+      server.addService(TestService.service, service)
+      server.start()
+
+      resolve(new TestService(`localhost:${port}`, grpc.credentials.createInsecure()))
```

Comment:
Can you change the bind calls to use `127.0.0.1` and change any reference of localhost to `127.0.0.1`? That should help with stability across different people's machines.